livre/lib/data/MemoryDataSource.cpp

```diff
@@ -88,21 +88,27 @@ MemoryUnitPtr MemoryDataSource::getData( const LODNode& node )
                             _volumeInfo.getBytesPerVoxel();
     const Identifier nodeID = node.getNodeId().getId();
     const uint8_t* id = reinterpret_cast< const uint8_t* >( &nodeID );
-    const uint8_t fillValue = ( id[0] ^ id[1] ^ id[2] ^ id[3] ) + 16 +
+    const uint8_t opacity =  ( id[0] ^ id[1] ^ id[2] ^ id[3] ) + 16 +
```

Comment:
Opacity comes with the mapping of that value to alpha with other N ( in RGB it is 3 ) elements. And the value itself does not show how high or low the opacity is. It is the index.
Even intensitity is a weak name. Because value of voxel can be interpreted to anything.
I dont know the best name, but those two are incorrect as well. To me voxelValue is enough abstract.